### PR TITLE
Revert "Revert "[DPTP-2094] ci-secret-bootstrap: validate if the given items/field/attachments exist""

### PR DESCRIPTION
**Pull request by droslean**
Reverts openshift/ci-tools#1859

/hold

### CODE REVIEW

**Comment by petr-muller** [comment]
Can't really review this: I don't know why this needed to be reverted, and why this is now safe to merge.

**Comment by droslean** [comment]
That broke the bootstrap validation presubmit, because it wasn't able to load the bitwarden credentials since it was running on a different cluster. There is some more digging to do. I am on it.

File: cmd/ci-secret-bootstrap/main.go [hunk 1]
```diff
@@ -777,6 +813,23 @@ func main() {
 	})
 	defer logrus.Exit(0)
 
+	if o.validateOnly {
```
Comment (alvaroaleman) [inline cmd/ci-secret-bootstrap/main.go:816]: This isn't fixed, `o.ValidateOptions` will still require kubeconfigs even though they are not needed.
/hold

Comment (droslean) [reply cmd/ci-secret-bootstrap/main.go:816]: Yea, I am on it. Thats why I kept this PR on hold

Comment (droslean) [reply cmd/ci-secret-bootstrap/main.go:823]: updated. Please check.

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**